website-custom-views/src/content/development/ui-components.mdx

```diff
@@ -11,12 +11,6 @@ To facilitate the development of UI features, we provide ready-to-use components
 
 The `<Spacings>` components let you position elements in your Custom View.
 
-<Info>
-
-To learn about structuring your page content, see [page content layouts](/development/page-content-layouts).
-
-</Info>
```

Comment:
Why was this removed?

Reply:
Hi @emmenko, I removed it because this page is no longer on the Custom Views website, but it's on Custom Applications, so I thought it was not relevant. Can you confirm if we should put the page as well as this Info box in the Custom Views docs?

Reply:
Hmm @CarlosCortizasCT shouldn't we have the `page-content-layouts` in the Custom Views docs as well?

Reply:
The idea is that users wouldn't be using these components in Custom Views as we would like for them to always use the whole available horizontal space.

We can have a discussion about this if you want to.

Reply:
@Anshuman71 after further discussion with UI/UX designer, we think we need to keep this content in the docs also for Custom Views.

Reply:
Thanks @CarlosCortizasCT. Then I'll add both the page and the info box back.

Reply:
Applied in fdf431b7ec79f20e56179d00922c1763a57f60bb